### ISSUE TITLE
cryptossh ReadLine() looks for "\r\n" cisco ios appears to use "\n"

### ISSUE THREAD

**Issue by tbotnz** [tbotnz/cisgo-ios#4]
noticed the return char not coming through when using netmiko

DEBUG:paramiko.transport:userauth is OK
INFO:paramiko.transport:Authentication (password) successful!
DEBUG:paramiko.transport:[chan 0] Max packet in: 32768 bytes
DEBUG:paramiko.transport:[chan 0] Max packet out: 32768 bytes
DEBUG:paramiko.transport:Secsh channel 0 opened.
DEBUG:paramiko.transport:[chan 0] Sesch channel 0 request ok
DEBUG:paramiko.transport:[chan 0] Sesch channel 0 request ok
DEBUG:netmiko:read_channel: cisgo1000v>
DEBUG:netmiko:read_channel: 
DEBUG:netmiko:read_channel: 
DEBUG:netmiko:read_channel: 
DEBUG:netmiko:write_channel: b'\n'
DEBUG:netmiko:read_channel: 
DEBUG:netmiko:read_channel: 
DEBUG:netmiko:write_channel: b'\n'
DEBUG:netmiko:read_channel: 
DEBUG:netmiko:write_channel: b'\n'

however if the below used it works

{
	"library": "netmiko",
	"connection_args": {
		"device_type": "cisco_ios",
		"host": "10.0.2.15",
		"username": "{{device_username}}",
		"password": "{{device_password}}",
		"port":10039,
		"fast_cli":true,
		**"default_enter":"\r\n"**
	},
	"command": "show ip interface brief",
	"queue_strategy": "fifo",
	"args":{
		"use_textfsm":true
	}
}

**Comment by wrgeorge1983** [comment]
That workaround works pretty well.  The Napalm variant is: 

```json
"connection_args": {
  "device_type": "cisco_ios",
  "host": "10.0.2.15",
  "username": "{{device_username}}",
  "password": "{{device_password}}",
  "optional_args": {
    "port": 10039,
    "fast_cli": true,
    "default_enter": "\r\n"
  }
},
```

**Comment by wrgeorge1983** [comment]
I know I've seen references to go commands specifically there to vendor libraries (`go get vendor` or something), so I assume that'd get us most of the way there

**Comment by tbotnz** [comment]
closed by #6
